sdk/core/core-client-rest/package.json

```diff
@@ -3,38 +3,32 @@
   "version": "1.2.1",
   "description": "Core library for interfacing with Azure Rest Clients",
   "sdk-type": "client",
-  "main": "dist/index.js",
-  "module": "dist-esm/src/index.js",
```

Comment:
since we're ditching `module` in favor of `exports["."]`, is this going to break any bundlers that work today? Or have we verified them against the new syntax?

Reply:
All modern bundlers should be using the correct [package.json resolution](https://nodejs.org/api/packages.html#package-entry-points), where this behavior was changed in Node 10.